stuff.py

```diff
@@ -152,10 +152,10 @@ async def on_voice_state_update(self ,member, before, after):
                     else:
                         await self.channel.send(ts+'***'+user+'*** hides himself in video call')
                 
-                #No afk channel policy
-                if after.channel != None:
-                    if after.channel.name == 'Pod mostem':
-                        await member.move_to(None,reason="OOF")
+        #No afk channel policy
+        if after.channel != None:
+            if after.channel == after.channel.guild.afk:
```

Comment:
Tested? Beacause i can doubt this can not work and i did not see a method there that works like this.

Reply:
upsie